pkg/server/server.go

```diff
@@ -60,11 +60,13 @@ type Dependencies struct {
 }
 
 type Config struct {
-	ResolveNodeLimit       uint32
-	ChangelogHorizonOffset int
-	ListObjectsDeadline    time.Duration
-	ListObjectsMaxResults  uint32
-	Experimentals          []ExperimentalFeatureFlag
+	ResolveNodeLimit              uint32
+	ChangelogHorizonOffset        int
+	ListObjectsDeadline           time.Duration
+	ListObjectsMaxResults         uint32
+	Experimentals                 []ExperimentalFeatureFlag
+	AllowWritingObsoleteModels    bool // Allow writing of model schema 1.0 even if it is being obsolete
```

Comment:
Similar to a previous comment, `AllowWritingObsoleteModels` is too ambiguous. Today, obsolete means 1.0 but in the future it could mean 1.0, 1.1,...

Reply:
updated